### PR TITLE
Content(UI): Move the title on the main menu up

### PR DESCRIPTION
**Pull request by Amazinite**
Someone commented on this and now it annoys me every time I see it.

Before:
image
image

After:
image
image

### CODE REVIEW

**Review by Saugia — APPROVED** [review]
yes

**Review by quyykk — APPROVED** [review]
Nice lol

**Review by roadrunner56 — APPROVED** [review]

**Review by Zitchas — APPROVED** [review]
Seems good. No unanticipated problems?

**Review by petervdmeer — APPROVED** [review]

**Review by Hurleveur — APPROVED** [review]
it seems like this needs more approvals before merging

**Review by quyykk — APPROVED** [review]

**Pull request merged**
